src/inference/enumerate.ad.js

```diff
@@ -19,6 +19,21 @@ module.exports = function(env) {
     options = util.mergeDefaults(options, {
       maxExecutions: Infinity
     });
+
+    this.throwOnError = options.throwOnError !== undefined ? options.throwOnError : true;
```

Comment:
I think you will be able to use the existing call to `mergeDefaults` (above) to more compactly specify the default value for `throwOnError`.